cla-backend/cla/models/github_models.py

```diff
@@ -1768,20 +1797,32 @@ def get_co_author_commits(co_author, commit, pr, installation_id):
     login, github_id = None, None
     email = co_author[1]
     name = co_author[0]
+
     # get repository service
     github = cla.utils.get_repository_service("github")
+
     cla.log.debug(f"{fn} - getting co-author details: {co_author}, email: {email}, name: {name}")
     try:
         user = github.get_github_user_by_email(email, installation_id)
     except (GithubException, IncompletableObject, RateLimitExceededException) as ex:
         # user not found
-        cla.log.debug(f"{fn} - co-author github user not found : {co_author} with exception: {ex}")
+        cla.log.debug(f"{fn} - co-author github user not found via email {email}: {co_author} with exception: {ex}")
         user = None
+    if user is None:
+        try:
+            # Note that Co-authored-by: name <email> is not actually a GitHub login but rather a name - but we are trying hard to find a GitHub profile
```

Comment:
And we need to do thsi kind of matching to find a real contributor's GitHub user. All we have is:
```
Co-authored-by: Name <email>
```

That name "seems to be" GitHub login in cases where I've checked, but technically it is "Name" not "Github login" - this comes from `git` commit data (trailer).

@mlehotskylf